oauth/api/utils.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from json import loads as load
+
+from fastapi import Security
+from fastapi.security import SecurityScopes
+from jose import jwe
+from jose.constants import ALGORITHMS
+from sqlalchemy.orm import Session
+
+from oauth.api.endpoints import TOKEN
+from oauth.api.exceptions import (
+    AuthenticationException,
+    AuthorizationException,
+    ExpiredTokenException,
+)
+from oauth.api.models import OAuth2ClientCredentialsBearer
+from oauth.database.models import ClientDetail
+import oauth.jwt as jwt
+from oauth.scopes import SCOPE_DOCS
+
+oauth2_scheme = OAuth2ClientCredentialsBearer(tokenUrl=TOKEN, scopes=SCOPE_DOCS)
```

Comment:
Nice, thank you for including the `scopes` here. That is a subtlety that I believe will generate some useful auto-docs in the Swagger API